tools/osbuilder/rootfs-builder/README.md

```diff
@@ -192,5 +192,4 @@ needed. Changes affect the files included in the final guest image.
 
 When building the rootfs for confidential containers if `SKOPEO=yes` is set then the `skopeo`
 package is built and added into the rootfs. It also adds the signature verification proof of concept files.
```

Comment:
You've left in 'It also adds the signature verification proof of concept files.' and `rootfs.sh` doesn't do that anymore ?

Reply:
Ah yeah good point - have removed that now, but left the info about skopeo and umoci